### PR TITLE
Logo and title break at medium width issue #91 fix

### PR DESCRIPTION
**Pull request by codesak**
Issue #91

### CODE REVIEW

File: website/src/index.css [hunk 1]
```diff
@@ -136,7 +136,7 @@ nav li li:last-of-type {
 }
 
 header strong {
-  font-size: 150%;
+  font-size: 120%;
```
Comment (Idrinth) [inline website/src/index.css:139]: please restore the size for bigger screen sizes

File: website/src/index.css [hunk 2]
```diff
@@ -166,12 +166,12 @@ a[target="_blank"] {
   }
 
   nav a {
-    font-size: 1.3em;
+    font-size: 1em;
```
Comment (Idrinth) [inline website/src/index.css:169]: please restore the size for bigger screen sizes

File: website/src/index.css [hunk 3]
```diff
@@ -166,12 +166,12 @@ a[target="_blank"] {
...
   }
 
   nav ul {
     flex-direction: row;
-    gap: 2em;
+    gap: 1.7em;
```
Comment (Idrinth) [inline website/src/index.css:174]: please restore the size for bigger screen sizes

**Review by Idrinth — CHANGES_REQUESTED** [review]
Thanks for the quick fix and your contribution, just some minor details :)

**Comment by codesak** [comment]
done, restore size for bigger screen

**Pull request merged**